src/deluge/processing/sound/sound_instrument.cpp

```diff
@@ -370,7 +370,7 @@ void SoundInstrument::sendNote(ModelStackWithThreeMainThings* modelStack, bool i
 
 		arpeggiator.noteOff(arpSettings, noteCode, &instruction);
 
-		if (instruction.noteCodeOffPostArp != ARP_NOTE_NONE) {
+		if (instruction.noteCodeOffPostArp != ARP_NOTE_NONE || noteCode == -32768) {
 
```

Comment:
change -32768 to ALL_NOTES_OFF

Reply:
didn't mean to include this so I'll take it out, this doesn't work and needs the arp itself to be modified